apps/wallet-mobile/src/features/Staking/Governance/common/SafeArea.tsx

```diff
@@ -0,0 +1,25 @@
+import React, {ReactNode} from 'react'
+import {StyleSheet} from 'react-native'
+import {SafeAreaView} from 'react-native-safe-area-context'
+
+import {COLORS} from '../../../../theme'
+
+type Props = {
+  children: ReactNode
+}
+
+export const SafeArea = ({children}: Props) => {
```

Comment:
that's ok, just a friendly reminder that `bottom` is controlled by `BottomTabNav`, if eventually part of the funnel display the bottom nav, this will have to change, dunno if worth it. 🤔 it will expose 1:1 to SAV.